src/pages/[platform]/build-a-backend/more-features/analytics/existing-resources/index.mdx

```diff
@@ -23,15 +23,19 @@ export function getStaticProps(context) {
 import android_existing_resources from '/src/fragments/lib/analytics/android/existing-resources.mdx';
 import flutter_existing_resources from '/src/fragments/lib/analytics/flutter/existing-resources.mdx';
 import ios_existing_resources from '/src/fragments/lib/analytics/ios/existing-resources.mdx';
-import js from '/src/fragments/lib/analytics/js/existing-resources.mdx';
-import react_native from '/src/fragments/lib/analytics/js/existing-resources.mdx';
+import js_existing_resources from '/src/fragments/lib/analytics/js/existing-resources.mdx';
+import react_native_existing_resources from '/src/fragments/lib/analytics/js/existing-resources.mdx';
```

Comment:
v small nit: both paths are the same, why not just use the `js_existing_resources` for RN as well?